runtime/queries/metricsview_timeseries.go

```diff
@@ -169,6 +169,11 @@ func (q *MetricsViewTimeSeries) Resolve(ctx context.Context, rt *runtime.Runtime
 			if v != nil {
 				t = *v
 			}
+		case int64:
+			if olap.Dialect() != drivers.DialectPinot {
+				panic(fmt.Sprintf("unexpected type for timestamp column: %T", v))
+			}
+			t = time.UnixMilli(v)
```

Comment:
Can we handle this in the driver using the result schema?

Reply:
again the issue is `DATETRUNC` returns LONG type in the schema instead of TIMESTAMP so unable handle generally.